docs/remote.md

```diff
@@ -49,6 +49,16 @@ When loaded, a remote driver process receives an HTTP POST on the URL `/Plugin.A
 
 Other entries in the list value are allowed; `"NetworkDriver"` indicates that the plugin should be registered with LibNetwork as a driver.
 
+### Set capability
+
+After Handshake, the remote driver will receive another POST message to the URL `/NetworkDriver.GetCapability` with no payload. The driver's response should have the form:
```

Comment:
minor:
just for future proofing we may want to name the endpoint `/NetworkDriver.GetCapabilities` to allow exchanging multiple capabilities, if and when libnetwork needs more of them.